tests/js/spec/stores/groupingStore.spec.jsx

```diff
@@ -483,6 +497,11 @@ describe('Grouping Store', function () {
       unmergeState = new Map([...GroupingStore.unmergeState]);
     });
 
+    afterEach(() => GroupingStore.teardown());
+
+    // WARNING: all the tests in this describe block are not running in isolated state.
+    // There is a good chance that moving them around will break them. To simulate an isolated state,
+    // add a beforeEach(() => GroupingStore.init()) and afterEach(() => GroupingStore.teardown()).
```

Comment:
Long term I think our plan should be to check Store interaction during tests and depending on state, auto-reset it, which means we'll eventually need to fix these, but for now 👍

Reply:
Yeah, I tried fixing these, but there's a lot going on and I couldn't get it to properly run. Def something we need to fix in the future, big 👍🏼